### ISSUE TITLE
refresh tree on demand without losing currently expanded nodes

### ISSUE THREAD

**Issue by galcho** [projectkudu/ARMExplorer#29]

**Comment by ahmelsayed** [comment]
I think this maybe tricky. We'll need to refresh the tree recursively for all the expanded levels, also correctly merge the data between the server and the client view.

**Comment by davidebbo** [comment]
Agreed that while a good idea, this feels a bit overambitious and we can leave that as a v2 kind of thing.
